build/webcodebook.js

```diff
@@ -1108,10 +1108,66 @@ function onResize$3() {
     //Annotate quantiles
     if (this.config.boxPlot) {
       var quantiles = [{ probability: 0.05, label: "5th percentile" }, { probability: 0.25, label: "1st quartile" }, { probability: 0.50, label: "Median" }, { probability: 0.75, label: "3rd quartile" }, { probability: 0.95, label: "95th percentile" }];
-
-      for (var item in quantiles) {
+      //console.log(quantile$$1);
+	  for (var item in quantiles) {
         var quantile$$1 = quantiles[item];
         quantile$$1.quantile = d3.quantile(this.values, quantile$$1.probability);
+		
+		// PLOT OUTLIERS
+		//--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+		if ( quantile$$1.probability == 0.05 ){
+			// probability of 0.05 corresponds to the lower whisker
+			var lower_whisker =  quantile$$1.quantile;
+			//Loop through all data points
+			for( var point in this.values ){
```

Comment:
`point` here references the position of each item in `this.values`.  To capture the value of that item use `array` notatation: `this.values[point]`.